Vienna/Sources/Preferences window/Preferences.m

```diff
@@ -268,8 +268,12 @@ -(NSDictionary *)allocFactoryDefaults
 	defaultValues[MAPref_AutoSortFoldersTree] = [NSNumber numberWithInt:VNAFolderSortManual];
 	defaultValues[MAPref_ShowFolderImages] = boolYes;
 	defaultValues[MAPref_UseJavaScript] = boolYes;
-    defaultValues[MAPref_UseNewBrowser] = boolNo;
-    defaultValues[MAPref_UseWebPlugins] = boolYes;
+	defaultValues[MAPref_UseNewBrowser] = boolNo;
+	if (@available(macOS 11, *)) {
+		defaultValues[MAPref_UseWebPlugins] = boolNo;
+	} else {
+		defaultValues[MAPref_UseWebPlugins] = boolYes;
+	}
```

Comment:
I would switch the default to No, whatever version of macOS is running, as this has been the default behavior of macOS and Safari for many years.